src/models/schema.ts

```diff
@@ -65,6 +65,7 @@ export class SchemaTreeItem extends vscode.TreeItem {
     const label = `v${resource.version}`;
     super(label, vscode.TreeItemCollapsibleState.None);
 
+    this.id = `${resource.id}-${resource.subject}-${resource.version}`;
```

Comment:
SchemaTreeItem ids need to be 'completely unique within the treeview', so that the treeview provider's implementation of getParent() and ability to highlight a specific treeitem will work.

Remember, cool cats and kittens, that a schema's 'id' only uniquely identifies its normalized document body, but that document may be bound to many subjects (and different versions) at a time within a single schema registry, but our Schema model object really repressents these bindings (and uh not the actual document body).